sdk/core/core-tracing/test/createSpan.spec.ts

```diff
@@ -29,86 +29,103 @@ describe("createSpan", () => {
     tracerProvider.disable();
   });
 
+  it("is backwards compatible at runtime", () => {
```

Comment:
After a year I will definitely not remember what versions is this intending to support :) some more context is appreciated.

Reply:
Hiii I added more context for this and all other places you mentioned (as well as everywhere else I am doing backwards compatibility at runtime)